components/admin_console/admin_definition.jsx

```diff
@@ -4795,7 +4797,14 @@ const AdminDefinition = {
             title: t('admin.sidebar.oauth'),
             title_default: 'OAuth 2.0',
             isHidden: it.any(
-                it.not(usesLegacyOauth),
+                it.any(
+                    it.not(it.licensed),
+                    it.all(it.licensedForSku('starter')),
```

Comment:
Can these be simplified to the following?
```suggestion
                    it.licensedForSku('starter'),
```